src/test/java/hw2/ex2/Exercise2.java

```diff
@@ -0,0 +1,58 @@
+package hw2.ex2;
+
+import hw2.base.TestBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+
+public class Exercise2 extends TestBase {
+
+    @Test
+    public void exerciseTwoTest() {
+        //1. Open test site by URL
+        openTestSite();
+        //2. Assert Browser title
+        SoftAssert softAssert = new SoftAssert();
```

Comment:
Why do you use soft asserts here?

Reply:
> Why do you use soft asserts here?

If I will use hard asserts in this test scenario that will break the rule "one test one assert", because of this I am using soft asserts here. With soft asserts, I am sure that all assets in the test will be checked, for hard assert if one assert failed, next would not be checked.

Reply:
For this task you should be use hard assert instead of soft assert.